### PR TITLE
Text modifications

### PR DESCRIPTION
**Pull request by jokudasai**

### CODE REVIEW

File: frontend/lib/pages/fee-waiver.tsx [hunk 1]
```diff
@@ -100,7 +100,7 @@ export const FeeWaiverExpenses = stepBuilder.createStep({
   </>,
   renderForm: ctx => <>
     <CurrencyFormField
-      label="How much do you pay in rent each month?"
+      label="How much do you pay in rent each month? If you live with someone else, please put in only the part of the rent that you pay."
```
Comment (toolness) [inline frontend/lib/pages/fee-waiver.tsx]: So this is also mentioned in the intro of the page, do you want it removed from there?  We also have the ability to put it in as a the hint text for a field, in which case it will be displayed below the field, un-bolded (I was worried that there might be too much bold text with it as part of the label).

File: frontend/lib/pages/hp-action-harassment.tsx [hunk 2]
```diff
@@ -22,7 +22,7 @@ export const HarassmentApartment = stepBuilder.createStep(props => ({
   </>,
   renderForm: ctx => <>
     <YesNoRadiosFormField {...ctx.fieldPropsFor('moreThanTwoApartmentsInBuilding')}
-      label="Are there more than 2 apartments in your building?" />
+      label="Does you building have 2 apartments or less?" />
```
Comment (toolness) [inline frontend/lib/pages/hp-action-harassment.tsx]: Hmm, so this is really a different question--the server/database stores this question as `moreThanTwoApartmentsInBuilding` but we really need to rename that, which will require a server-side change, so can you revert this text and we'll do these changes in a separate PR?

Comment (toolness) [reply frontend/lib/pages/hp-action-harassment.tsx]: Ok so I filtered all our users by who has answered this question already:

```pycon
>>> from hpaction.models import *
>>> deets = HarassmentDetails.objects.exclude(more_than_two_apartments_in_building=None)
>>> [deet.user.full_name for deet in deets]
['Jane Doe', '[REDACTED]', '[REDACTED]', 'Jane Doe', 'Jane Doe', 'Blargg Jones', 'Jane Doe', 'Blrerg Bones', 'Jane Doe', 'jane doe', 'jane doe']
```

So it looks like a bunch of dummy test users (who we should really delete) and two real people whose names I've redacted since this is a public forum.  That means we will have to first add a new model field called `two_or_less_apartments_in_building` and manually set the answers of those two people to the opposite of what they answered for the other question, and then delete the other question.

Comment (toolness) [reply frontend/lib/pages/hp-action-harassment.tsx]: Ok, I filed https://github.com/JustFixNYC/tenants2/pull/739 to deal with this one separately.

Comment (toolness) [reply frontend/lib/pages/hp-action-harassment.tsx]: Oops lol I was looking at the dev instance there.  We have no users who have filled out harassment info on production, so no need to migrate anything!

File: frontend/lib/pages/fee-waiver.tsx (hunk 1 above)
Comment (jokudasai) [reply frontend/lib/pages/fee-waiver.tsx]: now i see it. sorry, it wasn't on the priority guide

File: frontend/lib/pages/hp-action-harassment.tsx (hunk 2 above)
Comment (jokudasai) [reply frontend/lib/pages/hp-action-harassment.tsx]: great. do you still want me to revert the change?

**Comment by jokudasai** [comment]
@toolness would love to review the PR process with this example next week

File: frontend/lib/pages/hp-action-harassment.tsx (hunk 2 above)
Comment (toolness) [reply frontend/lib/pages/hp-action-harassment.tsx]: Oh yeah, I fixed this probalo in a different PR, so uh... honestly at this point it might just be easiest for you to close this PR, and make a new one based on the latest version of the code...
